docs/DeveloperGuide.md

```diff
@@ -2583,81 +2583,109 @@ Expected: The GUI closes and the application exits.
+          <br><br>
+    * **Test case:** `deleteExam 1`<br>
+      **Expected:** First exam is deleted from the address book. Status message shows the exam deleted.
+      <br><br>
+    * **Test case:** `deleteExam 0`<br>
+      **Expected:** No exam is deleted. Error message shown. No change in the address book.
+      <br><br>
+    * **Test case (index out of bounds):** `deleteExam 2`<br>
+      **Expected:** No exam is deleted. Error message shown. No change in the address book.
+      <br><br>
+    * **Test case (no index):** `deleteExam`<br>
+      **Expected:** No exam is deleted. Error message shown. No change in the address book.
 
 <br>
 
 <div id="test_selectexam"></div>
 
 #### Selecting an Exam: `selectExam`
 
-1. Prerequisites: Exactly one exam in the address book. Hence, exam has an index of 1.
-
-2. Test case: `selectExam 1`<br>
-   Expected: First exam is selected. Status message shows the exam selected.
+**Command:** `selectExam`<br>
+**More information on usage:** <a href="UserGuide.md#selectexam">Selecting an Exam</a>
 
-3. Test case: `selectExam 0`<br>
-   Expected: No exam is selected. Error message shown. No change in the address book.
+1. Selecting an exam
 
-4. Test case: `selectExam 2` (index out of bounds)<br>
-   Expected: No exam is selected. Error message shown. No change in the address book.
+    * **Prerequisites:** 
+        * Exactly one exam in the address book. Hence, exam has an index of 1.
+          <br><br>
+    * **Test case:** `selectExam 1`<br>
+      **Expected:** First exam is selected. Status message shows the exam selected.
+      <br><br>
+    * **Test case:** `selectExam 0`<br>
+      **Expected:** No exam is selected. Error message shown. No change in the address book.
+      <br><br>
+    * **Test case (index out of bounds):** `selectExam 2`<br>
+      **Expected:** No exam is selected. Error message shown. No change in the address book.
+      <br><br>
+    * **Test case (no index):** `selectExam`<br>
+      **Expected:** No exam is selected. Error message shown. No change in the address book.
 
-5. Test case: `selectExam` (no index)<br>
-   Expected: No exam is selected. Error message shown. No change in the address book.
+<br>
 
 <div id="test_deselectexam"></div>
 
 #### Deselecting an Exam: `deselectExam`
 
-1. Prerequisites: An exam has been selected.
+**Command:** `deselectExam`<br>
+**More information on usage:** <a href="UserGuide.md#deselectexam">Deselecting an Exam</a>
 
-2. Test case: `deselectExam`<br>
-   Expected: Selected exam is deselected. Status message shows the exam deselected.
+1. Deselecting an Exam
 
-3. Test case: `deselectExam` (no exam selected)<br>
-   Expected: No exam is deselected. Error message shown. No change in the address book.
+    * **Prerequisites:** 
+        * An exam has been selected.
+          <br><br>
+    * **Test case:** `deselectExam`<br>
+      **Expected:** Selected exam is deselected. Status message shows the exam deselected.
```

Comment:
not sure if you want to state this but can talk about the highlighting of the exam because when you deselect the exam the exam should no longer be highlighted